### ISSUE TITLE
CallLogging, custom requestToLogString

### ISSUE THREAD

**Issue by fraadsbrandth** [ktorio/ktor#1047]
### Ktor Version
1.1.3

### Ktor Engine Used(client or server and name)
netty server

### Feedback
Currently it logs the request on format below, but i would also like to log the query parameters.
```
fun ApplicationRequest.toLogString(): String = "${httpMethod.value} - ${path()}"
```
One solution would be to add a function variable in the Configuration object and call for this function wherever the "ApplicationRequest.toLogString()" is currently called.

Something like this, that defaults to the current format

```
var requestToString : (ApplicationRequest) -> String = { it.toLogString() }
```

Or is it any way with the current implementation to accomplish the same?

**Comment by cy6erGn0m** [comment]
It should be easy to implement your own logging feature with any format except that you probably may need MDC access that makes it much more difficult to implement.

**Comment by fraadsbrandth** [comment]
Yes, in that case I would use CallLogging feature only for MDC support, add a filter predicate to it to ensure the feature does not **log** any requests. And further create my own CallLogging feature logging on the format of my choice. It just seems a bit strange..

**Comment by cy6erGn0m** [comment]
Fixed in 1.2.0-rc2
